### PR TITLE
Updated notebook cypress tests

### LINKED ISSUE #628: [BUG] Observability Flaky Cypress tests
**Describe the bug**
The cypress tests are flaky. They do not run successfully consistently. 

**To Reproduce**
Run all the cypress tests for 10 times. The major errors come in App Analytics and Panels. 

**Expected behavior**
The flakiness comes into picture when the tests wait for buttons to clicked or inputs are mistyped by cypress.

### PR DESCRIPTION
**Pull request by ps48**
Signed-off-by: Shenoy Pratik <sgguruda@amazon.com>

### Description
updated notebook cypress tests, added data cleanup and updated eui 2.0 toggle button clicks

### Issues Resolved
#628 

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by kavithacm — APPROVED** [review]

**Review by kavithacm — COMMENTED** [review]
LGTM!

File: dashboards-observability/.cypress/integration/notebooks.spec.js [hunk 1]
```diff
@@ -27,17 +39,22 @@ describe('Adding sample data and visualization', () => {
...
     cy.get('.euiModalHeader__title[data-test-subj="confirmModalTitleText"]')
       .contains('Add samples')
       .should('exist');
-    cy.get('.euiButton__text').contains('Yes').click();
-    cy.wait(delay * 5);
-    cy.get('.euiTableCellContent').contains(SAMPLE_PANEL).should('exist');
+    cy.wait(100);
+    cy.get('.euiButton__text').contains('Yes').trigger('mouseover').click();
+    cy.wait(100 * 5);
+    cy.route2('POST', '/addSamplePanels').as('addSamples');
```
Comment (joshuali925) [inline dashboards-observability/.cypress/integration/notebooks.spec.js:53]: why is this needed?

**Review by joshuali925 — APPROVED** [review]

File: dashboards-observability/.cypress/integration/notebooks.spec.js (hunk 1 above)
Comment (ps48) [reply dashboards-observability/.cypress/integration/notebooks.spec.js:53]: This is needed to load sample observability data. The data is used to add a sample observability visualization in notebooks.
